File: src/main/java/me/yamakaja/commanditems/data/CommandItemsConfig.java
package me.yamakaja.commanditems.data;

import java.util.Map;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * Created by Yamakaja on 26.05.18.
 */
public class CommandItemsConfig {

    @JsonProperty("items")
    private Map<String, ItemDefinition> items;

    public Map<String, ItemDefinition> getItems() {
        return this.items;
    }

}

File: src/main/kotlin/me/relaxing9/commanditems/data/CommandItemsConfig.kt
package me.relaxing9.commanditems.data

import com.fasterxml.jackson.annotation.JsonProperty

/**
 * Created by Yamakaja on 26.05.18.
 */
class CommandItemsConfig {
    @JsonProperty("items")
    val items: Map<String, ItemDefinition>? = null
}